studiopress/atomic-blocks
# Add logo list section design

**Summary of change:**
This PR adds the logo list section design to the layout modal.

**This PR has been:**

**How to test:**
Add the layout block, select the Logo List section and visually test it.

**See:** #186 

**Suggested Changelog Entry:**
Add Logo List section.
